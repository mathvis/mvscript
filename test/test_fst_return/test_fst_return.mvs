func a() bool {
	return false;
}
