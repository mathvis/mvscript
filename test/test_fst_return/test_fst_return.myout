(Decl (FunctionDef (FunctionIdentifier "a") [] BoolT (Just (Block (FunctionBlock BoolT) [Expr (Return (Just (Type (Bool False))))]))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = BoolT, arguments = [], hasBody = True})], context = Nothing})
