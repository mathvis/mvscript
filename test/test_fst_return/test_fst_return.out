(Decl (FunctionDef (FunctionIdentifier "a") [] BoolT (Just (Block (FunctionBlock BoolT) []))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = BoolT, arguments = [], hasBody = True})], context = Nothing})
